python/test/t_Gibbs_std.py

```diff
@@ -1,14 +1,55 @@
 #! /usr/bin/env python
-
 import openturns as ot
 import openturns.testing as ott
-import math as m
+import math
 
 ot.TESTPREAMBLE()
 
 ot.RandomGenerator.SetSeed(0)
 
-# this analytical example is taken from "Bayesian Modeling Using WinBUGS" - Ioannis Ntzoufras
+# Test the Metropolis within Gibbs algorithm to simulate a correlated 2D Gaussian target :
+
+# Define the target law
+mean_target = [0.0, 0.0]
+std_target = [1.0, 1.0]
+rho_target = 0.9
+target_normal = ot.Normal( mean_target, std_target, ot.CorrelationMatrix([[1., rho_target],[rho_target, 1.]]) )
```

Comment:
I suggest to use Black to fix this.
```suggestion
target_normal = ot.Normal(mean_target, std_target, ot.CorrelationMatrix([[1.0, rho_target], [rho_target, 1.0]]) )
```

Reply:
Done.